### PR TITLE
Bump libgit2 to 0.28.2

### PR DESCRIPTION
**Pull request by nalimilan**
This is the companion to https://github.com/JuliaLang/julia/pull/32806. I hope that's correct.

### CODE REVIEW

File: LibGit2/build_tarballs.jl [hunk 1]
```diff
@@ -1,20 +1,19 @@
 using BinaryBuilder
 
 name = "LibGit2"
-version = v"0.27.7"
+version = v"0.28.2"
 
 # Collection of sources required to build Ogg
```
Comment (ararslan) [inline LibGit2/build_tarballs.jl]:
```suggestion
# Collection of sources required to build libgit2
```
haha

**Comment by staticfloat** [comment]
lgtm

**Comment by staticfloat** [comment]
https://github.com/JuliaPackaging/Yggdrasil/releases/tag/LibGit2-v0.28.2%2B0

**Comment by nalimilan** [comment]
Thanks. Looks like the build scripts expect the release to be called "-1" rather than "+0"? At least that's my interpretation of the failures at https://githvub.com/JuliaLang/julia/pull/32806.

**Comment by staticfloat** [comment]
Yes, we are transitioning from a `vX.Y.Z-BUILD` format to `vX.Y.Z+BUILD` format, as it makes version comparisons simpler.  (We don't actually do any version comparisons yet, but someday we will).

You need to change this line: https://github.com/JuliaLang/julia/pull/32806/files#diff-a01ad0cac7aa5a648c5d00ec39bd9afcR100

Instead of `LibGit2-v$(LIBGIT2_VER)-$(LIBGIT2_BB_REL)` at the end, change it to `LibGit2-v$(LIBGIT2_VER)+$(LIBGIT2_BB_REL)`.  Then also change `LIBGIT2_BB_REL` to `0` instead of `1`.

**Pull request merged**